src/views/dialogs/ComplianceConfigDialog.tsx

```diff
@@ -41,7 +41,8 @@ export const usePreferenceMenu = () => {
       items: complianceStatusOptions.map(({ status, label }) => ({
         value: status,
         label: label,
-        onSelect: () => dispatch(setCompliance({ geo, status } as Compliance)),
+        onSelect: () =>
+          dispatch(setCompliance({ geo, status, updatedAt: new Date().toString() } as Compliance)),
```

Comment:
omega-small-nit: Instead of saving date as string, we could also use `Date.now()`. String implementation is completely fine since it would be used exactly the same (`new Date(complianceUpdatedAt`)

Reply:
I did it this way because, I wanted it emulate the response indexer is giving back, since it is a formated ISOstring and not a timestamp.